Command/RunParatestCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Liip\FunctionalTestBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+/**
+ * Command used to update the project.
+ */
+class RunParatestCommand extends ContainerAwareCommand
+{
+    private $container;
+    private $configuration;
+    private $output;
+    private $process = 5;
+    private $testDbPath;
+    private $phpunit = './bin/phpunit';
+
+    /**
+     * Configuration of the command.
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('test:run')
+            ->setDescription('Run phpunit tests with multiple process')
+        ;
+    }
+
+    protected function prepare()
+    {
+        $this->configuration = $this->getContainer()->hasParameter('liip_functional_test');
+        $paratestCfg = (!isset($this->configuration['paratest'])) ? array('process' => $this->process, 'phpunit' => $this->phpunit) : $this->configuration['paratest'];
+
+        $this->process = (!empty($this->configuration['process'])) ? $paratestCfg['process'] : $this->process;
+        $this->phpunit = (!empty($this->configuration['phpunit'])) ? $paratestCfg['phpunit'] : $this->phpunit;
+        $this->testDbPath = $this->getContainer()->get('kernel')->getRootDir();
+        $this->output->writeln("Cleaning old dbs in $this->testDbPath ...");
+        $createDirProcess = new Process('mkdir -p '.$this->testDbPath.'/cache/test/');
+        $createDirProcess->run();
+        $cleanProcess = new Process("rm -fr $this->testDbPath/cache/test/dbTest.db $this->testDbPath/cache/test/dbTest*.db*");
```

Comment:
it s just to be sure you don t test with an old cloned Db ( for example if you adjust a fixtures for a tests, or if one of your test alter Db data such as new entity created, next time you ll have a constraint exception or an unprocessable entity ) .